src/extensions/attributes/index.js

```diff
@@ -30,36 +29,23 @@ function addAttributes( settings ) {
  * @param {function|Component} BlockEdit Original component.
  * @return {string} Wrapped component.
  */
-const withAttributes = createHigherOrderComponent( ( BlockEdit ) => {
-	return ( props ) => {
-		const {
-			attributes,
-			setAttributes,
-		} = props;
+const withAttributes = createHigherOrderComponent(
+	BlockEdit => props => {
+		const { name: blockName } = props;
```

Comment:
This bit is just a simple refactor of the nested callbacks to improve readability.